Geometry2d/testing/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+#======================================================================
+# Set Testing Sources
+#======================================================================
```

Comment:
Shouldn't this also be in the parent?

Reply:
I think this is a style thing

Reply:
Okay, I'm fine with it. Just different from what I've seen - it does make the `soccer` CMakeLists more readable, though.

Reply:
Yeah. It's something I saw in my internship and I think I like it better than what I've been writing.